### ISSUE TITLE
A way of hiding tags you added

### ISSUE THREAD

**Issue by Yimellovzski** [kurozenzen/r34-react#61]
**Describe the feature**
When the list of tags added gets too long, it can take a while to scroll all the way down to press the search button to update to show new posts, doing this every time i want to update posts gets tedious so perhaps (since this webapp is designed with mobile phones in mind)  a way to hide all the tags you inputted could be implemented, if possible.

**Comment by kurozenzen** [comment]
Good point. I will look into it. Probably I will make so it only displays one line of tags and indicates that there are more. And then once you click on then it opens up and shows all of them.

How does that sound to you?

**Comment by Yimellovzski** [comment]
Sure it sounds like a good idea, maybe make it toggle-able so it doesn't hide the tags that will get hidden if they hit the limit threshold and only are a few (one or two).
